src/Snowcap/Emarsys/Client.php

```diff
@@ -250,17 +250,24 @@ public function createContact(array $data)
      * Updates one or more contacts/recipients, identified by an external ID.
      *
      * @param array $data
+     * @param bool $createIfNotExists When enabled, if the contact does not exist in the database, it is created
+     * automatically.
      * @return Response
+     * @link http://documentation.emarsys.com/resource/developers/endpoints/contacts/update-contacts/
```

Comment:
Done. 

But that was a link to official API documentation. So I thought it's nice to use just `Shift+F1` in my PhpStorm when I'm confusing, for example, what I should pass in `$data`.